bindings/pyroot/pythonizations/python/ROOT/_pythonization/_rdataframe.py

```diff
@@ -388,6 +389,17 @@ def __call__(self, *args):
 
         return res
 
+def _PyFilter(rdf, func, funcargs = {}, filter_name = ""):
+    """
+    Filters the RDF according to the function or string provided.
```

Comment:
Perhaps it would be better to explain what this pythonization is for. The goal here is to support the use of Python callables as an argument of Filter. Internally, the pythonization takes care of jitting such callable with Numba with the right parameter types, which are taken from the types of the dataset columns used in the Filter.